ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -89,7 +87,7 @@ export class WorkflowsList extends BasePage<RouteComponentProps<any>, State> {
             },
             namespace: this.props.match.params.namespace || '',
             selectedPhases: this.queryParams('phase').length > 0 ? this.queryParams('phase') : savedOptions.selectedPhases,
```

Comment:
do we need to the same for phases?

Reply:
I think phases makes more sense to store for two reasons: (1) it's a close-ended input (we provide all available options), and (2) I think the use case makes more sense